SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocol.m

```diff
@@ -28,6 +28,7 @@ @interface SDLProtocol () {
     dispatch_queue_t _sendQueue;
     SDLPrioritizedObjectCollection *_prioritizedCollection;
     NSMutableDictionary *_sessionIDs;
+    NSMutableDictionary *_hashIDs;
```

Comment:
Just tried to best following existing naming so when we do changes it'll be consistent ;)